adapter/stackdriver/metric/merge.go

```diff
@@ -107,19 +108,43 @@ func mergePoints(a, b *monitoring.TimeSeries) (*monitoring.TimeSeries, error) {
 		}
 		a.Points[0].Value = &monitoring.TypedValue{&monitoring.TypedValue_DoubleValue{av.DoubleValue + bv.DoubleValue}}
 	case *monitoring.TypedValue_DistributionValue:
-		if _, ok = b.Points[0].Value.Value.(*monitoring.TypedValue_DistributionValue); !ok {
+		var bv *monitoring.TypedValue_DistributionValue
+		if bv, ok = b.Points[0].Value.Value.(*monitoring.TypedValue_DistributionValue); !ok {
 			return a, fmt.Errorf("can't merge two timeseries with different value types; a is a distribution, b is not: %#v", b.Points[0].Value)
 		}
-		return a, fmt.Errorf("not implemented")
-		// TODO: combining distributions is hard. We know that they have the same buckets since they're all for the same metric, so that part is fine.
-		// But we need to update counts of each bucket, as well as the mean, sum of squared deviation, and the range. This is non-trivial.
+		// TODO: in theory we should assert that the DistributionValue's options are identical before merging, but given
```

Comment:
would this be worth logging as a warning, if detected ?

Reply:
I didn't want to bother checking it at runtime - we can add the check and return an error. I just don't really think it's necessary.